### PR TITLE
Setup repo

### PR DESCRIPTION
**Pull request by tthyer**
* Copies most setup from packer-rstudio and packer-base-ubuntu-bionic
* Creates a "hello world" playbook

### CODE REVIEW

**Review by thomasyu888 — APPROVED** [review]

**Comment by thomasyu888** [comment]
Ah, I would set up precommit to fix all the small syntax issues.

**Comment by tthyer** [comment]
Thanks, I always forget

File: src/template.json [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+      "output": "manifest.json",
+      "strip_path": true,
+      "type": "manifest"
+    }
+  ],
+  "provisioners": [
+    {
+      "ansible_env_vars": [
+        "ANSIBLE_SSH_ARGS='-o IdentitiesOnly=yes'",
+        "JcConnectKey={{user `JcConnectKey`}}"
```
Comment (zaro0508) [inline src/template.json]: you don't need this.

**Review by zaro0508 — CHANGES_REQUESTED** [review]

File: src/template.json (hunk 1 above)
Comment (tthyer) [reply src/template.json]: removed

File: src/template.json [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  "post-processors": [
+    {
+      "output": "manifest.json",
+      "strip_path": true,
+      "type": "manifest"
+    }
+  ],
+  "provisioners": [
+    {
+      "ansible_env_vars": [
```
Comment (zaro0508) [inline src/template.json:42]: your missing the ImageName.  it needs to look like this.. https://github.com/Sage-Bionetworks/packer-base-ubuntu-bionic/blob/master/src/template.json#L42

**Review by zaro0508 — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -1,2 +1,82 @@
...
+* Request an IAM account in [Imagecentral](https://github.com/Sage-Bionetworks/imagecentral-infra)
+* Change password and set up MFA
+* Create a Keypair
+* Add your access code and secrety key to `~/.aws/credentials`, using a profile such as "imagecentral.jsmith"
+* Authenticate with `awsmfa`, for example `awsmfa -i imagecentral.jsmith -t jsmith@imagecentral`
+* Finally, add the following
+```
+[profile packer-service-imagecentral]
+region = us-east-1
+role_arn = arn:aws:iam::867686887310:role/accounts-PackerServiceRole-18QJ34FLREC6E
```
Comment (zaro0508) [inline README.md]: might be good to not provide the real role_arn?

**Review by zaro0508 — CHANGES_REQUESTED** [review]

File: src/template.json (hunk 2 above)
Comment (tthyer) [reply src/template.json:42]: @zaro0508 Do I need that? It does not appear that way in the [packer-rstudio template](https://github.com/Sage-Bionetworks/packer-rstudio/blob/master/src/template.json#L42).
I was able to do a manual build without that.

File: README.md (hunk 3 above)
Comment (tthyer) [reply README.md]: fixed and overwrote commit

**Review by zaro0508 — APPROVED** [review]

File: src/template.json (hunk 2 above)
Comment (zaro0508) [reply src/template.json:42]: no, i finally figured it out that we pass it in with a packer -var flag

**Pull request merged**